unlock-app/src/__tests__/components/creator/CreatorLockForm.test.js

```diff
@@ -212,7 +213,7 @@ describe('CreatorLockForm', () => {
       expect(wrapper.getByValue('0.01').dataset.valid).toBe('true')
     })
     it('submit button is enabled and activates on submit', () => {
-      const wrapper = makeLockForm()
+      const wrapper = makeLockForm({ convert: true })
```

Comment:
And same here: something along the lines of this is a legacy test which has a different data format...

Reply:
quick note: this is actually the only non-legacy test, the one that proves the new way works, but I will add a note to remove the convert once the new testing way is figured out